test-case/verify-ucm-bootsequence.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+##
+## Case Name: verify-ucm-bootsequence.sh
+## Preconditions:
+##    N/A
+## Description:
+##    Run alsactl init to test the initialization settings are correct or not
+## Case step:
+##    1. check the alsa-lib, alsa-utils version
+##    2. save the original amixer settings
+##    3. run alsactl init
+##    4. restore the original amixer settings
+## Expect result:
+##    alsactl init runs successfully
+##
+
+set -e
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+# BootSequence is introduced in alsa-utils 1.2.2-16
+ALSAUTILS_VER=1.2.2-16
+# BootSequence is introduced in alsa-lib 1.2.2-44
+ALSALIB_VER=1.2.2-44
+
+func_opt_parse_option "$@"
+
+check_alsa_lib_ver()
+{
+    alsalib_ver=$(dpkg -l libasound2 | grep libasound2 |awk '{print $3}')
```

Comment:
will all OS use dpkg? What about user build bianry?

Reply:
This is not supported because we can't accurate versions of alsa-lib if user uses build binary.